jpower432/compliance-trestle-fedramp
# feat: populate template with control origination

## Types of changes




## Summary

Stacked with #9 

### How to test

1. Create a virtual environment
```bash
python -m venv .venv
. .venv/bin/activate
```
3. Install `compliance-trestle`
```
pip install compliance-trestle
```
4. Clone with this branch
5. Install with `make install`
6. Make a directory adjacent to the cloned repo and run `trestle init`
7. Import the test SSP
```bash
trestle import -f compliance-trestle-fedramp/tests/data/json/simplified_fedramp_ssp_template.json -o test-ssp
```
6. Run the plugin command
```bash
trestle fedramp-transform -l high --ssp-name test-ssp -o /data/fedramp_test.docx
```
7. Open the document and scroll to the AC-1 Control Summary. Observe the checked box.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)
